JeffShepherd/fitness-tracker
# Initial setup

## What does this PR do?

- Adds reset CSS file
- Add boilerplate class and class test files
- PR template

## How should it be tested in the UI?
- Run 'npm start' and ensure there are no errors in the browser, and the title of the application displays
- Run 'npm test' to ensure the boilerplate test for each class runs and passes
